<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="4">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>航班名称</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>航班编号</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLineEdit" name="flight_name"/>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="start"/>
    </item>
    <item row="2" column="5">
     <widget class="QLineEdit" name="end"/>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="flight_no"/>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>起始地</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>目的地</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="7">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QPushButton" name="add_flight">
        <property name="text">
         <string>添加航班</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_client">
        <property name="text">
         <string>添加客户</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="query_record">
        <property name="text">
         <string>查询订单</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="query_client">
        <property name="text">
         <string>查询客户</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="7">
     <widget class="QPushButton" name="query_flight">
      <property name="text">
       <string>查询航班</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="7">
     <widget class="MyTableView" name="table"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTableView</class>
   <extends>QTableView</extends>
   <header>table.py</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
